.gitignore

```diff
@@ -49,6 +49,7 @@ tools
 
 _artifacts
 _build
+_test
```

Comment:
When tests clean up after themselves, we don't need this. But when they don't, it's a Git mess